pino.js

```diff
@@ -121,7 +122,7 @@ function asJson (obj, msg, num) {
   // to catch both null and undefined
   /* eslint-disable eqeqeq */
   if (msg != undefined) {
-    data += ',"msg":' + JSON.stringify('' + msg)
+    data += ',"' + this.messageProp + '":' + JSON.stringify('' + msg)
```

Comment:
can we just generate the encoded version, like we had ',"msg":' before?

Also, the messageProp would need to be encoded for " and the like.